src-docs/src/views/suggest_item/suggest_item_example.js

```diff
@@ -77,7 +77,7 @@ export const SuggestItemExample = {
       ],
       text: (
         <p>
-          By default <EuiCode>EuiSuggestItem</EuiCode>'s{' '}
+          By default <EuiCode>EuiSuggestItem</EuiCode>&apos;s{' '}
```

Comment:
Writing text in the docs is weird sometimes. This just uses the unicode code for apostrophe so we don't have to escape it.